website/docs/r/external_network.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_external_network"
+sidebar_current: "docs-vcd-resource-external-network"
+description: |-
+  Provides a vCloud Director external network resource.  This can be used to create and delete external networks.
+---
+
+# vcd\_external\_network
+
+Provides a vCloud Director external network resource.  This can be used to create and delete external networks.
+
+Supported in provider *v2.2+*
+
+## Example Usage
+
+```hcl
+resource "vcd_external_network" "net" {
+  name        = "my-ext-net"
+  description = "Reference for VDC direct network"
+
+  ip_scope {
+    is_inherited = "false"
+    gateway      = "192.168.30.49"
+    netmask      = "255.255.255.240"
+    dns1         = "192.168.0.164"
+    dns2         = "192.168.0.196"
+    dns_suffix   = "mybiz.biz"
+
+    static_ip_pool {
+      start_address = "192.168.30.51"
+      end_address   = "192.168.30.62"
+    }
+  }
+
+  vsphere_networks {
+    vcenter         = "vC1"
+    vsphere_network = "myNetwork"
+    type            = "DV_PORTGROUP"
+  }
+
+  fence_mode                         = "isolated"
+  retain_net_info_across_deployments = "false"
+}
+
+resource "vcd_network_direct" "net" {
+  name             = "my-net"
+  external_network = "${vcd_external_network.net.name}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the network
+* `description` - (Optional) Network friendly description
+* `ip_scope` - (Required) A list of IP scopes for the network.  See [IP Scope](#ipscope) below for details.
+* `vsphere_networks` - (Required) A list of DV_PORTGROUP or NETWORK objects that back this network. Each referenced DV_PORTGROUP or NETWORK must exist on a vCenter server registered with the system.  See [vSphere Networks](#vspherenetworks) below for details.
+* `fence_mode` - (Optional) Isolation type of the network. If ParentNetwork is specified, this property controls connectivity to the parent. One of: bridged (connected directly to the ParentNetwork), isolated (not connected to any other network), natRouted (connected to the ParentNetwork via a NAT service) 
+* `retain_net_info_across_deployments` - (Optional)  Specifies whether the network resources such as IP/MAC of router will be retained across deployments. Default is false.
+* `parent_network` - (Optional) Contains reference to parent network
+
+
+<a id="ipscope"></a>
+## IP Scope
+
+* `is_inherited` - (Required) True if the IP scope is inherit from parent network
+* `gateway` - (Required) Gateway of the network
+* `netmask` - (Required) Network mask
+* `dns1` - (Required) Primary DNS server
+* `dns2` - (Required) Secondary DNS server
```

Comment:
This should be optional. It is not required in the GUI

Reply:
Changed too